convene-web/config/locales/en.yml

```diff
@@ -61,6 +61,27 @@ en:
   guides:
     title: "Guides"
     call_to_action: "Learn How"
-    locked_rooms:
-      title: "Locked Rooms"
-      value_proposition: "Get some privacy, require a passcode to enter a room."
\ No newline at end of file
+    spaces:
+      title: "Spaces"
+      value_proposition: "Lay out a Space of your own!"
+    rooms:
+      title: "Rooms"
+      value_proposition: "Bring people and information together!"
+    furniture:
+      title: "Furniture"
+      value_proposition: "Activities, Tools, and Toys for your Rooms!"
+    neighborhoods:
+      title: "Neighborhoods"
+      value_proposition: "You're not alone! Explore!"
+    hookups:
+      title: "Hookups"
+      value_proposition: "Connect your Space to other websites or online services!"
+    people:
+      title: "People"
```

Comment:
Noodling a bit on the friendly/welcoming aspect of this, what if we had an emoji go with each title? E.g:
Guides 🧭 
Spaces 🪐 
Rooms 🛋️ 
...

Reply:
I hope "hookups" isn't confusing/misleading in the table of contents, without the "utilities" context. 🤔

Reply:
A valid point :X.